apps/yara/mapper.h

```diff
@@ -223,6 +225,11 @@ struct MapperTraits
     typedef String<CigarElement<> >                                 TCigar;
     typedef StringSet<TCigar, Segment<TCigar> >                     TCigarSet;
     typedef StringSetLimits<TCigarSet>::Type                        TCigarLimits;
+    typedef Position<TCigarSet>::Type                               TCigarSetPos;
```

Comment:
To avoid further confusion, please rename cigar-based types as follows:

```
    typedef String<CigarElement<> >                                 TCigarString;
    typedef StringSet<TCigarString, Segment<TCigarString> >         TCigars;
    typedef StringSetLimits<TCigars>::Type                          TCigarsLimits;
    typedef Position<TCigars>::Type                                 TCigarsPos;
    typedef String<TCigarsPos>                                      TCigarsPositions;
    typedef ModifiedString<TCigars, ModPos<TCigarsPositions> >      TCigarsView;
    typedef StringSet<TCigars, Owner<ConcatDirect<> > >             TCigarsSet;
```